consent/manager_memory.go

```diff
@@ -473,7 +473,7 @@ func (m *MemoryManager) ListUserAuthenticatedClientsWithFrontChannelLogout(ctx c
 	return rs, nil
 }
 
-func (m *MemoryManager) ListUserAuthenticatedClientsWithBackChannelLogout(ctx context.Context, subject string) ([]client.Client, error) {
+func (m *MemoryManager) ListUserAuthenticatedClientsWithBackChannelLogout(ctx context.Context, subject, sid string) ([]client.Client, error) {
```

Comment:
You have added the `sid` parameter here, but are not actually using it. It should do the same thing that you're doing in the SQL adapter!

Reply:
If I remove it, go will say that the consent manager   incorrectly implements `ListUserAuthenticatedWithBackChannelLogout`

Reply:
Do not remove it, you need to add the same check as you did in the SQL manager.

Reply:
you want me to check with the `sid` parameter too right